cordova/cordova-sampleapp-android-tests/sampleapp/Crosswalk_Cordova_SampleApp_Eh_AppFunction.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<!--
+Copyright (c) 2015 Intel Corporation.
+
+Redistribution and use in source and binary forms, with or without modification,
+are permitted provided that the following conditions are met:
+
+* Redistributions of works must retain the original copyright notice, this list
+  of conditions and the following disclaimer.
+* Redistributions in binary form must reproduce the original copyright notice,
+  this list of conditions and the following disclaimer in the documentation
+  and/or other materials provided with the distribution.
+* Neither the name of Intel Corporation nor the names of its contributors
+  may be used to endorse or promote products derived from this work without
+  specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY INTEL CORPORATION "AS IS"
+AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ARE DISCLAIMED. IN NO EVENT SHALL INTEL CORPORATION BE LIABLE FOR ANY DIRECT,
+INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,
+EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+Authors:
+    Zhu,Yongyong <yongyongx.zhu@intel.com>
+
+-->
+
+<html>
+  <head>
+    <meta charset='utf-8'>
+    <title>Crosswalk_Cordova_SampleApp_Eh_AppFunction</title>
+    <link rel="author" title="Intel" href="http://www.intel.com">
+  </head>
+  <body>
+    <h4>Note</h4>
+    <ol>
+      <li>It's only for Cordova version 4.0</li>
+    </ol>
+    <p>
+      <strong>Preconditions:</strong>
+    </p>
+    <ol>
+      <li>A good network which is able to access to google service</li>
+      <li>Two google accounts used to contact each other</li>
+      <li>Two android devices which are able to access to google service and signed in with the above google accounts</li>
+      <li>Install 'Eh' sample app in the two android devices(Refer to 'README' file)</li>
+      <li>One host connected to the network to run GCM server</li>
+      <li>Install python in the host</li>
+      <li>Put GCM server(https://github.com/MobileChromeApps/workshop-cca-eh/tree/master/gcmServer) in the host</li>
+      <li>Set 'username' with '585199982707' and set 'password' with 'AIzaSyCXxKFxOTwdkp5I6V1858TF_5d0m4wk9Ps' in gcmServer/gcm_auth_info.json</li>
+      <li>Start the GCM server using 'python gcmServer/python/server.py'</li>
+    </ol>
+    <p>
+      <strong>Test steps:</strong>
+    </p>
+    <ol>
+      <li>Launch 'Eh' sample app in the first android device</li>
+      <li>An authorization prompt might appear, then click "OK" link to allow app access the GCM server</li>
```

Comment:
it is to allow app and Google to use the information of your google account rather than allow app to access the GCM server.
s/link/button